### ISSUE TITLE
flymake-ruff not reporting the issues from ruff check if wrong identation

### ISSUE THREAD

**Issue by pmercatoris** [erickgnavar/flymake-ruff#34]
```sh
~ ❯ emacs --version
GNU Emacs 30.2
Copyright (C) 2025 Free Software Foundation, Inc.
GNU Emacs comes with ABSOLUTELY NO WARRANTY.
You may redistribute copies of GNU Emacs
under the terms of the GNU General Public License.
For more information about these matters, see the file named COPYING.

~ ❯ ruff --version
ruff 0.13.1
```

`~/minimal-debug.el`:
```emacs-lisp
(require 'package)
(add-to-list 'package-archives '("melpa" . "https://melpa.org/packages/"))
(package-initialize)

(unless (package-installed-p 'use-package)
  (package-refresh-contents)
  (package-install 'use-package))
(require 'use-package)

(use-package flymake
  :ensure nil
  :hook (prog-mode . flymake-mode))


(use-package flymake-ruff
  :ensure t
  :hook (python-mode . flymake-ruff-load)
  )
```

<img width="637" height="457" alt="Image" src="https://github.com/user-attachments/assets/bc81e7e0-ca71-4b2e-9445-ac30554b9c40" />

when adding a symbol at the wrong place in the buffer, it just doesn't report anything anymore, even though `ruff check` reports everything.

<img width="685" height="863" alt="Image" src="https://github.com/user-attachments/assets/8686a72f-5f6f-4981-b0d5-84e8a0c21b23" />

again here with wrong indentation

<img width="699" height="579" alt="Image" src="https://github.com/user-attachments/assets/63c68836-c2f6-4f2e-8d67-634012f0da17" />

**Comment by vimmoos** [comment]
@pmercatoris  I had the same issue. I made a PR to fix it. 
In the meantime, the simple workaround is to change the flymake-ruff--output-regex  and add the invalid-syntax to the severity map (otherwise it defaults to :warning) 
My current setup is: 
``` emacs-lisp
  (use-package flymake-ruff
    :ensure t
    :config
    (setq flymake-ruff--output-regex
        "\\(.*\\):\\([0-9]+\\):\\([0-9]+\\): \\([A-Za-z0-9-]+\\):? \\(.*\\)")
    (add-to-list 'flymake-ruff--severity-map '("invalid-syntax" . :error)))
```
